### DISCUSSION TITLE
Make the remote redux devtool work for physical devices or emulators

### DISCUSSION THREAD

**Discussion by ZahraRahmani** [ngrx/platform#3508] [Q&A]
### Information

Is there a way to set up redux devtool for android/ios physical devices? Running an app with ionic serve will show the state and actions on the browser, I was looking for a way to make it work for the actual devices or emulators. I have been following https://gist.github.com/rob3c/c2c4dcc1116f94901ace179722c5f6d4 and https://medium.com/nextfaze/remote-debugging-ngrx-store-with-ionic-74e367316193 instructions to make the remote redux devtool show the actions and states for the physical device and/or emulators for debugging, I got everything setup, but was not seeing any actions or states coming to the remote redux devtool. I was not able to find other articles or a clear guidance on how to make this happen. Looking forward to hear any information or guidance that can help me with this. Thank you.

Application information:
rxjs: 6.6.7
typescript: 4.4.4
Angular CLI: 13.0.1
Node: 16.16.0

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

**Comment by timdeschryver** [comment]
I'm converting this to a discussion because this is more a question than a bug/feature.

**Comment by BenGrn** [comment]
This is a needed feature! I managed to get the blog/gist working however it is very clunky and missing all typing. The library required has not been updated since 2016 and requires modification within the node_modules path which is not practical.

**Reply by PubuduS** [reply]
Hello Ben,

Could you please tell me how you did that? I also tried blog and gist but couldn't figure out how to do that.
